composer.json

```diff
@@ -45,9 +45,14 @@
         "apigen": [
             "php devtools/vendor/bin/apigen -c build/config/apigen.neon -d build/api -s src -s vendor/symfony/config -s vendor/symfony/console -s vendor/symfony/yaml -s vendor/guzzle/guzzle/src -s vendor/psr/log --report build/logs/checkstyle-apigen.xml --exclude *Test.php --exclude */Tests/* --google-analytics UA-40398434-1"
         ],
+        "install-box": [
+            "curl -LSs http://box-project.github.io/box2/installer.php | php",
```

Comment:
please raise it as an issue to Box project.
It would even try to use autoloader for composer.json in root location of partition (eg C:\ on Windows) if the composer.json file exists.